src/Symfony/Component/Form/Resources/translations/validators.en.xlf

```diff
@@ -14,6 +14,10 @@
                 <source>The CSRF token is invalid. Please try to resubmit the form.</source>
                 <target>The CSRF token is invalid. Please try to resubmit the form.</target>
             </trans-unit>
+            <trans-unit id="99">
```

Comment:
`31`?

Reply:
It must not override another id of the Validator validators.*.yaml files since it's the same domain.